### DISCUSSION TITLE
Filter Timestamp UTC Button

### DISCUSSION THREAD

**Discussion by amsterdamn2023** [tchiotludo/akhq#1495] [Q&A]
Hi guys,

I was browsing through the update offsets menu and came across a button labeled "Filter Timestamp UTC Button." What is the function of this button? Does it allow resetting an offset to a specific date and time? If so, how does it work?

If not, is this something you plan to implement in the near future?

Thanks

**Comment by tchiotludo** [comment]
Yes this button allow to reset consumer group offset to this specify timestamp, using internal native kafka api [offsetsForTimes](https://github.com/tchiotludo/akhq/blob/d898b462217a8069ae172f796812d172dd3e84f6/src/main/java/org/akhq/repositories/RecordRepository.java#L190)

**Reply by amsterdamn2023** [reply]
Can you tell me what I need to do with the internal native Kafka API OffsetsForTimes? This is for an internship assignment, so I need to have a clear understanding of it. 

Does this feature work by default or do I need to make any modifications/configurations?

Thank you in advance for your response!
